src/sth_configuration.js

```diff
@@ -1,14 +1,21 @@
-/* globals module, process */
+/* globals module, process, require */
 
 (function() {
   "use strict";
 
+  var path = require('path');
+
   var ENV = process.env;
 
   var dbUsername = ENV.DB_USERNAME || '';
   var dbPassword = ENV.DB_PASSWORD || '';
 
   module.exports = {
+    LOG_LEVEL: ENV.LOG_LEVEL || 'info',
+    LOG_TO_CONSOLE: ENV.LOG_TO_CONSOLE !== 'false',
```

Comment:
I understand this ('LOG_TO_CONSOLE`) and next (`LOG_TO_FILE`) are the default values... If both of them are false, that means by default nothing is logged? Maybe one of them (probably console) should be at least true... what do you think?